.changeset/proud-onions-yawn.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/features": minor
+"@wso2is/console": minor
```

Comment:
Since the changes in this PR doesn't add a new feature, let's bump the packages to a patch version.

```suggestion
"@wso2is/features": patch
"@wso2is/console": patch
```